spec/tree_spec.rb

```diff
@@ -1,14 +1,94 @@
 require 'rspec'
-require 'tree'
+require_relative '../lib/tree'
 
-describe 'Tree' do
+describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(described_class.is_a? Class).to be_truthy
```

Comment:
The be_truthy matcher works here, but `expect(described_class.is_a? Class).to be true` could be better since we don't need to match on all truthy values. Using an rspec type matcher would be better still, e.g.: `expect(described_class).to be_a Class`. This is a little bit of a nitpick but I think it would help readability. [RSpec Type Matchers](https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/type-matchers)